### ISSUE TITLE
Missing `data` value for create action

### ISSUE THREAD

**Issue by LucianBuzzo** [olivierwilkinson/prisma-nested-middleware#25]
In v2, you could find field values in nested create action under `params.args.data`. As of v3, when doing a nested create, these field values are directly under `params.args`.  As a result I have to write logic to handle both cases, by looking at the `scope` value. e.g. 

```
const data = params.scope ? params.args : params.args.data;
```

This change isn't mentioned in the v3 release notes, so I'm presuming that it's unintentional.

**Comment by olivierwilkinson** [comment]
Hi there,

Yes you are right it's not in the release notes for v3, I'm sorry! I clearly forgot to mention it in the breaking changes, or at all it looks like. I'm not planning any other major architectural changes so I don't expect to need another big commit (🤞) but I will make sure to be more methodical in the future regardless.

In terms of the actual change: it was intentional, the original idea for moving the `args` from a nested create to `params.args.data` was to keep the top level schema the same for all create actions, however I realised that there were many other actions that would need to do this, and not all of them could be mapped to the root args schema for their action type. In the end I decided that it was easier to be consistent and not transform the args rather than introducing a whole new set of rules. It also became difficult to determine what schema to restore the nested action args to after executing the middleware.

I've updated the release notes to mention this and added a link to this issue.

Sorry again, I hope it didn't cause you any major problems!
